README.md

```diff
@@ -728,6 +728,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Frazier Odhiambo](https://github.com/frazie)
   - [Furkan Mutlu](https://github.com/furkanmutlu)
   - [Furqan Abid](https://github.com/heyyfurqan)
+  - [Frazier Mark](https://github.com/FrazierMark)
```

Comment:
Hey @FrazierMark, Please add your name in alphabetical order! 

Please let us know if you need any further help:)

Reply:
Oh! Thanks for the correction. I've made the change. thx